### PR TITLE
wrong base address fix.

### PR DESCRIPTION
**Pull request by koviroli**
## Summary

Base adress code of How To Create Wcf Client page, is not correct. Maybe the VB script also incorrect.

### CODE REVIEW

File: docs/framework/wcf/how-to-create-a-wcf-client.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The client application uses the generated proxy class to communicate with the se
     ```
 
     ```shell
-    svcutil.exe /language:cs /out:generatedProxy.cs /config:app.config http://localhost:8000/GettingStartedClient/service
+    svcutil.exe /language:cs /out:generatedProxy.cs /config:app.config http://localhost:8000/GettingStarted/CalculatorService
```
Comment (Thraka) [inline docs/framework/wcf/how-to-create-a-wcf-client.md]: The `svcutil` call above for VB should also use this fixed URL.

Comment (Thraka) [inline docs/framework/wcf/how-to-create-a-wcf-client.md]: It would be great if you could fix the intro URL also. It should not use `GettingStartedClient`

**Review by Thraka — CHANGES_REQUESTED** [review]
@koviroli Thank you for opening this PR. Could you fix the other two error spots?

**Comment by koviroli** [comment]
Ok I fix it

**Review by Thraka — APPROVED** [review]

**Comment by Thraka** [comment]
@koviroli thank you!!

**Pull request merged**